### DISCUSSION TITLE
does V7 support Markers plugin?

### DISCUSSION THREAD

**Discussion by logesh2496** [katspaugh/wavesurfer.js#2994] [Q&A]
I have been looking for markers plugin example with v6 and couldn't find any. Does V7 have support for it? could you also share the old examples link of v6?

**Comment by katspaugh** [comment]
In v7, you can just use Regions. A region without an end point is a marker.

**Comment by katspaugh** [comment]
V6 Markers docs: https://wavesurfer-js.org/plugins/markers

**Comment by logesh2496** [comment]
thanks, How can I produce these markers at the top using regions?

<img width="1485" alt="image" src="https://github.com/katspaugh/wavesurfer.js/assets/55389513/685123e6-3cad-4169-9165-cddf5f51beb4">

**Reply by logesh2496** [reply]
<img width="21" alt="image" src="https://github.com/katspaugh/wavesurfer.js/assets/55389513/6988a409-3d08-40cc-b881-bb98683bb54d">

**Reply by katspaugh** [reply]
You can pass a custom element in the content property of a region. Or use custom CSS targeting the `::part(region)` selector.
See https://wavesurfer-js.org/examples/#styling.js
